### ISSUE TITLE
User selectable manual audio delay

### ISSUE THREAD

**Issue by 5225225** [TeamNewPipe/NewPipe#11200]
### Checklist

- [X] I made sure that there are *no existing issues* - [open](https://github.com/TeamNewPipe/NewPipe/issues) or [closed](https://github.com/TeamNewPipe/NewPipe/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I have read the [FAQ](https://newpipe.net/FAQ/) and my problem isn't listed.
- [X] I'm aware that this is a request for NewPipe itself and that requests for adding a new service need to be made at [NewPipeExtractor](https://github.com/TeamNewPipe/NewPipeExtractor/issues).
- [X] I have taken the time to fill in all the required details. I understand that the feature request will be dismissed otherwise.
- [X] This issue contains only one feature request.
- [X] I have read and understood the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/dev/.github/CONTRIBUTING.md).

### Feature description

A user selectable audio delay that is tied to the Bluetooth device in use. Probably allow just typing in a (positive or negative, but delaying the video to sync up with audio is more useful for me) millisecond value.

### Why do you want this feature?

My Bluetooth headphones have around 200ms of latency on them, and when I watch videos with them on, its quite jarring. (A secondary use which would be to fix audio sync on a per-video basis)

(I've heard there's a feature to let headphones *tell* the phone what delay they have, but either this is not implemented for my headphones, or it needs player support which doesn't exist (yet))

### Additional information

VLC has this feature and it works well in my experience, I just set a audio delay and tell VLC to always use that delay for my given Bluetooth headphones. (It uses no delay when using phone speakers).

The UI looks like this (but I wouldn't copy the "sound heard / sound seen" personally, I would just allow the user to type in a delay and expect them to figure out the correct delay).

![Screenshot_20240622-004946](https://github.com/TeamNewPipe/NewPipe/assets/8584210/393404b1-788e-4392-a1e3-7b2ee5a74e3b)

And as requested by the template, here's a picture of a cat I saw a while back :3
![markup_4110](https://github.com/TeamNewPipe/NewPipe/assets/8584210/f0b05267-c756-4bcc-8d01-5653702c4c8f)

**Comment by 5225225** [comment]
Actually... I tested it again with my current headphones and it does seem like the latency *is* automatically adjusted. Maybe it's a newer firmware thing?

I think the feature request is still valid though, since I'm not sure if *all* headphones support it, but it's at least not too important for me personally anymore.

**Comment by jakariyaa** [comment]
Hello! I have this exact issue with YouTube/NewPipe. My earbuds also have ~200ms and it's really annoying to watch videos with lips out of sync. I can get away with this latency by setting a video delay in MX player (also in VLC player) it would be really nice to similar option in NewPipe. So please reopen this issue.

![Bluetooth audio delay option](https://github.com/user-attachments/assets/51a2e3d3-32b8-4a7d-855f-de599a627d93)
An screenshot of Mx Player having this Bluetooth audio delay option

**Comment by jakariyaa** [comment]
> @jakariyaa Do you get a lag using https://f-droid.org/en/packages/com.brouken.player/ ?

Yes @opusforlife2, I still get this audio lag with this mentioned player.

Tested version: https://github.com/moneytoo/Player/releases/tag/v0.170

**Comment by jakariyaa** [comment]
@opusforlife2  I have opened this similar issue in the repo of NewPlayer: 
https://github.com/TeamNewPipe/NewPlayer/issues/9

I hope it was the right place.
